content/en/kb/butterfly-network/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "How to join the Butterfly network."
```

Comment:
Can we make it say Butterfly (test). We need to make is very clear that this is a test network.